modules/dr-restoring-cluster-state.adoc

```diff
@@ -9,6 +9,14 @@
 
 You can use a saved etcd backup to restore back to a previous cluster state. You use the etcd backup to restore a single control plane host. Then the etcd cluster Operator handles scaling to the remaining master hosts.
 
+[IMPORTANT]
+====
+When restoring, you must use an etcd backup from the same z-stream release as the current cluster. For example, an {product-title} 4.5.2 cluster must use an etcd backup that was taken from 4.5.2.
+
+// TODO: Can we just leave it at that, or do we need to add an additional statement like below?
+// If you are attempting to restore your cluster using an etcd backup that was taken from a previous z-stream release, contact Red Hat Support for [best effort assistance?] [???].
+====
```

Comment:
cc @smarterclayton  @sdodson @sferich888 : my understanding was that as with rollback customer is to contact support for "this is not really supported but we want to help you situation". I wanted to make sure we are all on the same page here.

We want a very clear message. Currently, we only support time trave within same Z. But if you are in trouble (IE backup is from previous version) contact support.

Reply:
Although it might be true that support will try to help you out for things like this, in the docs [1] we have a clear "rollbacks are unsupported" message:

```
Reverting your cluster to a previous version, or a rollback, is not supported. Only upgrading to a newer version is supported.
```
That's what makes me hesitate on putting something in the docs to say to contact support for something we want to say is unsupported.

[1] https://docs.openshift.com/container-platform/4.5/updating/updating-cluster-between-minor.html#update-service-overview_updating-cluster-between-minor

Reply:
Sure but we should say if you are having issues with an upgrade please contact support?